test/orbit_web/controllers/react_app_controller_test.exs

```diff
@@ -6,7 +6,9 @@ defmodule OrbitWeb.ReactAppControllerTest do
     conn = get(conn, ~p"/")
     response = html_response(conn, 200)
     assert response =~ "Orbit"
-    # sets release
+
+    # sets metadata
     assert response =~ "<meta name=\"release\" content=\"test\">"
+    assert response =~ "<meta name=\"userEmail\" content=\"user@example.com\">"
```

Comment:
**suggestion:** can you also test that the relevant Sentry `<meta/>` tags are present?

Reply:
I actually tried, but putting test Sentry configuration in `test.exs` causes backend Sentry to try to load and connect to those fake resources that don't exist. I could mock `get_env` itself here, but that seems kinda weird. Thoughts?

Reply:
Would something like [this macro](https://github.com/mbta/skate/blob/main/test/support/helpers.ex#L4) that we've copy-pasted to a few repos help? It basically sets the variable while also setting up an `on_exit` to clean up the value so that it doesn't interfere with other tests.

Reply:
Ooh! Like. I'll give it a shot.

Reply:
If the meta tag is based on `Application.get_env(:orbit, :environment)`, (from my other comment) then the meta tag could be set even without turning on sentry.

Reply:
fixed in d463f99